src/components/TableComplex/RemoveDeviceDialog.react.js

```diff
@@ -31,10 +31,10 @@ class RemoveDeviceDialog extends Component {
   };
   // Handle changes in device name
   handleChange = event => {
-    this.setState({
+    this.setState((state, props) => ({
```

Comment:
was the change made to this file necessary?

Reply:
to fix the alert, yes :) [details](https://lgtm.com/rules/1819283066/)